include/stdshims.h

```diff
@@ -1,19 +1,25 @@
 //! Shims for std:: functions that aren't available in the current C++ versions
 //! we target.
 
-#pragma once
+#ifndef STDSHIMS_H
+#define STDSHIMS_H
 
 #include <memory>
 #include <utility>
 
 #if (__cplusplus >= 201402L)
 #warning "This file should now be removed! The functions it provides are part of the C++14 standard."
-#endif
+using std::unique_ptr;
```

Comment:
As I mentioned before, this should be `using std::make_unique;`.

Reply:
Ah, my bad - thanks for catching that. Fixed.